alembic/versions/09f4d0a776b0_add_column_is_shop_available.py

```diff
@@ -0,0 +1,42 @@
+"""add column is_shop_available
+
+Revision ID: 09f4d0a776b0
+Revises: 30c3d93498d1
+Create Date: 2020-06-25 22:11:39.027201
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '09f4d0a776b0'
+down_revision = '30c3d93498d1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    #op.drop_table('spatial_ref_sys')
+    op.drop_index('idx_address_location', table_name='address')
+    op.add_column('shops', sa.Column('is_shop_available', sa.Boolean(), nullable=True))
+    op.drop_index('idx_shops_location', table_name='shops')
```

Comment:
Comment out `drop_index` statements. We just want to add new column.

Reply:
ok